config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -280,6 +284,11 @@ public ServerHttpSecurity securityContextRepository(ServerSecurityContextReposit
 		return this;
 	}
 
+	public HttpsRedirectSpec redirectToHttps() {
```

Comment:
Please add some javadoc. An example of how you might look for a header only in production so that redirects always happen in production might be useful.